tensorboard/components/tf_ng_tensorboard/app.module.ts

```diff
@@ -48,4 +52,15 @@ import {ReloaderModule} from './reloader/reloader.module';
   providers: [],
   bootstrap: [AppComponent],
 })
-export class AppModule {}
+export class AppModule {
+  constructor(
+    private domSanitizer: DomSanitizer,
+    iconRegistry: MatIconRegistry
+  ) {
+    iconRegistry.addSvgIconSet(this.getIconUrl());
+  }
+
+  private getIconUrl() {
+    return this.domSanitizer.bypassSecurityTrustResourceUrl(SVG_PATH);
```

Comment:
Ah, so we need to explicitly bypass any URL to make it a SafeResourceURL that Angular trusts while sanitizing